tomviz/LoadDataReaction.cxx

```diff
@@ -256,6 +276,65 @@ DataSource* LoadDataReaction::loadData(const QStringList& fileNames,
   return dataSource;
 }
 
+bool LoadDataReaction::loadTiffStack(const QStringList& fileNames,
+                                     std::vector<ImageFileInfo>& summary)
+{
+
+  vtkSmartPointer<vtkTIFFReader> reader = vtkSmartPointer<vtkTIFFReader>::New();
```

Comment:
Should prefer vtkNew for this, avoids duplication/long line.

Reply:
Fixed in https://github.com/OpenChemistry/tomviz/pull/1535/commits/9883950e6f70d5dd20dc79458084289fec5fdd7f